mono/metadata/class.h

```diff
@@ -39,6 +39,10 @@ mono_class_from_name_case  (MonoImage *image, const char* name_space, const char
 MONO_API MonoMethod *
 mono_class_get_method_from_name_flags (MonoClass *klass, const char *name, int param_count, int flags);
 
+MonoMethod *
+mono_class_get_method_from_name_checked (MonoClass *klass, const char *name,
```

Comment:
Plumbed through MonoError upon request. Consider "flags" in the name.

Reply:
Not need to line-break here.